cpg-core/specifications/language.md

```diff
@@ -0,0 +1,33 @@
+# Specification: Language and Language Frontends
```

Comment:
How do we make sure that people will find this documentation? Having multiple locations for documentation could be confusing...
- Include a hint in the readme file?
- and / or include this (somehow) in the generated documentation

Reply:
Both good ideas. In the long run I want to add this to the generated doc. In short I’ll add a link in the readme

Reply:
I would move this to the wiki itself if the wiki allows rendering of the graphs, and then make links from the main read.me to the wiki, which should already be tere

Reply:
This discussion is probably outside of the scope of the PR but I would actually prefer a good generated documentation on some github.io page instead of the wiki that also includes these specifications, but let's discuss this separately.